controller_manager/include/controller_manager/controller_manager.h

```diff
@@ -138,7 +138,7 @@ class ControllerManager{
    */
   bool switchController(const std::vector<std::string>& start_controllers,
                         const std::vector<std::string>& stop_controllers,
-                        const int strictness);
+                        const int strictness, bool start_asap = false, double timeout = 0.0);
```

Comment:
`timeout` could be better named, also please update the docstring with some description of the parameters

Reply:
Description updated.